executor/seqtest/seq_executor_test.go

```diff
@@ -941,7 +941,7 @@ func TestBatchInsertDelete(t *testing.T) {
 		atomic.StoreUint64(&kv.TxnTotalSizeLimit, originLimit)
 	}()
 	// Set the limitation to a small value, make it easier to reach the limitation.
-	atomic.StoreUint64(&kv.TxnTotalSizeLimit, 5500)
+	atomic.StoreUint64(&kv.TxnTotalSizeLimit, 5600)
```

Comment:
Why do we need to change this value?

Reply:
struct tikvTxn contains model.TableInfo, and we add TiFlashMode in TableInfo. Thus we need to expand the size limit, to avoid be tooLargeTxn.